wsd-frontend/src/components/wsd/Notifications/Notifications.tsx

```diff
@@ -60,20 +62,64 @@ export function Notifications({ hasNew }: { hasNew?: boolean }) {
     }
   }
 
+  async function markAllAsRead() {
```

Comment:
I think we can just have a single markAllAsRead async function that we pass to onClick `onClick={markAllAsRead}`

And instead of using callbacks can you use async/await syntax to keep things more consistent when possible?